services/dask-sidecar/src/simcore_service_dask_sidecar/settings.py

```diff
@@ -50,4 +50,4 @@ def as_worker(self) -> bool:
     @validator("LOG_LEVEL", pre=True)
     @classmethod
     def _validate_loglevel(cls, value: Any) -> str:
-        return cls.validate_log_level(value)
+        return cast(str, cls.validate_log_level(str(value)))
```

Comment:
```suggestion
        return cast(str, cls.validate_log_level(f"{value}"))
```

Reply:
@sanderegg you should explain @matusdrobuliak66 why you do this ;-)

Reply:
Here is one reason, but also the f-string I find more readable.
https://medium.com/@DahlitzF/pythons-f-strings-vs-str-e22995cefef6

Reply:
Ok thank you for the tip, I will add it.